### ISSUE TITLE
2.0 BiTDI Firmware (3G8907159)

### ISSUE THREAD

**Issue by MasterTommyNL** [jilleb/mqb-soundaktor#23]
Hey folks!

First of all, kudo's for all the work that has been done here. I threw a donation in as a first thing. 

Installed the soundaktor on my Tiguan AD1 2018 (2.0 BiTDI) a year ago, only to run into the 'wrong vehicle type' code and no joy. 
The tiguan didn't come with a soundaktor, also not in other configurations, but the Arteon with the same engine did. So i have a soundaktor from the Arteon.

Suddenly found out this website, and i tried flashing another firmware that was unlocked (Golf GTD), but also no joy. Soundaktor code disappeared, soundaktor is happy but does not receive a driving profile message from the gateway, so i'll have to get an original dataset for that at some point and modify it, also to put the soundaktor setting in the individual driving profile settings menu. 

But the gateway dataset is a secondary issue. First and foremost i was wondering the following:

I see firmware with part numbers that were, to my knowledge, never available as flash files. So my question is; were these read from the modules themselves? I have another 3G8907159 module laying around with the original flash still in there. If the firmware can be read, that would be great so it will allow me to keep it as stock as it gets. 

Long story short: 

1: Any way at all to get the 3G8907159 flash file? No problem to ship a module to someone either if they can extract firmware, and of course not for free because going through the trouble for me is worth something. The reason i want this specific firmware is because the engine is quite a bit different from other TDI's and i have driven an Arteon with the same engine and soundaktor, and it was perfect. Not too quiet, not too loud, just exactly right all the way.

2: Any way someone can pull an original gateway dataset for my VIN? Unmodified so i have something to tinker with. Again, no problem at all compensating someone for the time it took.  

Many thanks for reading!

**Comment by MasterTommyNL** [comment]
I’m stupid, i just realized the 3G0 firmware was also featured in the passat with this engine. So 3G0 would also work. Ill have a look to see if i can find the corresponding engine type (CUAA) in the flash file data.
